fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -365,6 +365,7 @@ public void getData(
                     uriInfo,
                     this
             );
+            RequestLog.stopTiming("DataApiRequest");
```

Comment:
I know why `switchTimer` isn't working. Here's the implementation:

```java
stopMostRecentTimer();
startTiming(nextPhase);
```
However, the `DataApiRequest` timer is no longer the most recent timer, because of all the timings we added to the `DataApiRequest`. So, `switchTimer` doesn't do what you think it should if you have timings happening inside any method calls between `startTimer` and `switchTimer`.